docs/build/guides/basics/classic-transition.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Shifting from Stellar Classic to smart contracts
```

Comment:
I'm not sure if this is the best title. Maybe "Add support for smart contracts"?